web_front/src/pages/Calendar.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import styles from './Calendar.module.css';
+import { callApi } from './api';
```

Comment:
You do realise that it doesnt even compile? Also add a new <Route> in App.js and add link to it to links in Header.js